### PR TITLE
Catch up latest docs: from 5.1 to 5.4

### PR DESCRIPTION
**Pull request by BlueCutOfficial**
This is an automatic catch up PR to align our non-translated documentation with the latest official documentation.

### CODE REVIEW

File: scripts/catchup.mjs [hunk 1]
```diff
@@ -0,0 +1,236 @@
...
+Please assign yourself to the issue or let a comment at the very moment you start the translation.
+        
+File: \`${filename}\`
+From Ember: **${currentEmberVersion}**
+To Ember: **${newEmberVersion}**
+        
+\`\`\`diff
+${diffblock}
+\`\`\`
+      `,
```
Comment (GreatWizard) [inline scripts/catchup.mjs:136]:
```suggestion
`,
```

File: scripts/catchup.mjs [hunk 2]
```diff
@@ -0,0 +1,236 @@
...
+}
+console.log('shelljs: "git reset --hard upstream/master" executed');
+
+if (shell.exec('git push origin -f ref-upstream').code !== 0) {
+  console.log('shelljs: "git push origin -f ref-upstream" command failed');
+  process.exit(1);
+}
+console.log('shelljs: "git push origin -f ref-upstream" executed');
+
+if (shell.exec('git switch master').code !== 0) {
```
Comment (GreatWizard) [inline scripts/catchup.mjs:232]: What about a shell/exec function like this and use it everywhere:

```js
function shellexec(command){
  if (shell.exec(command).code !== 0) {
    throw new Error(`shelljs: "${command}" command failed`);
  }
  console.log(`shelljs: "${command}" executed`);
}
```

File: scripts/catchup.mjs [hunk 3]
```diff
@@ -0,0 +1,236 @@
...
+  } catch (error) {
+    console.log('Issue posting has failed:', error);
+    issuePostingError = true;
+  }
+  await new Promise(resolve => setTimeout(resolve, 1000));
+});
+
+if (!issuePostingError) {
+  // Once the issues are posted, delete patches folder and files
+  fs.rmSync('scripts/patches', { recursive: true, force: true });
```
Comment (GreatWizard) [inline scripts/catchup.mjs:165]: Should be better with a main try/catch and then do the rmSync in the finally

**Review by GreatWizard — APPROVED** [review]

**Comment by BlueCutOfficial** [comment]
=> #220

**Pull request merged**